TicTacToeApp/ViewController.swift

```diff
@@ -8,18 +8,38 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+public class ViewController: UIViewController {
+  var currentPlayerMark = "X"
 
-  override func viewDidLoad() {
+  override public func viewDidLoad() {
     super.viewDidLoad()
     // Do any additional setup after loading the view, typically from a nib.
   }
 
-  override func didReceiveMemoryWarning() {
+  override public func didReceiveMemoryWarning() {
     super.didReceiveMemoryWarning()
     // Dispose of any resources that can be recreated.
   }
 
+  @IBOutlet public var boardButtons: [UIButton]!
+  @IBOutlet weak var resetButton: UIButton!
 
+  @IBAction public func resetBoard() {
```

Comment:
resetGame() seems reasonable - I also have it in mind to try to extract some methods from it's current form. I'll add this rename to my list of low hanging fruit.